sqrl-testing/sqrl-integration-tests/src/test/java/com/datasqrl/flink/FlinkPhysicalIT.java

```diff
@@ -74,21 +73,36 @@ public void tableColumnDefinitionTest() {
   }
 
   @Test
-  public void jsonToString() {
+  public void jsonInStreamTest() {
+    jsonTest(false);
+  }
+
+  @Test
+  public void jsonInDatabaseTest() {
+    jsonTest(true);
+  }
+
+  public void jsonTest(boolean inDatabase) {
```

Comment:
Both of these are executed in Flink right? One is on stream and the other on state?

Reply:
no, one is executed in flink and the other in postgres

Reply:
Ill use the database hint to make it more explicit